clang/test/3C/unwritable_rootcauses.h

```diff
@@ -0,0 +1,15 @@
+
```

Comment:
Ideally, we would test that the "unwritable code" reason takes priority over other reasons for wildness for all of the constructs in the `clang/test/3C/root_cause.c` test, not just the few you chose for this file.  As I mentioned in #664, you may be able to do this by just having `clang/test/3C/base_subdir/unwritable_rootcauses.c` do `#include "../root_cause.c"`, specify `-Xclang -verify=unwritable-expected-warning`, and then add `// unwritable-expected-warning` comments to `clang/test/3C/root_cause.c`.  That has the additional advantage of ensuring that we're testing the same constructs that do actually have other root causes of wildness when they are in writable code.

Reply:
I'm misunderstanding how the diagnostic verifier works,  won't those comments interface with the test that uses `root_cause.c`? That was my plan originally but I didn't know a way around that.

Reply:
To handle this scenario, the diagnostic verifier lets you specify a custom comment prefix to `-verify`; then it will only recognize expected-diagnostic comments with that prefix.  So `root_cause.c` can specify `// RUN: ... -Xclang -verify` (which is short for `-Xclang -verify=expected`) and use `// expected...` comments, and `unwritable_rootcauses.c` can specify `// RUN: ... -Xclang -verify=unwritable-expected` and use `// unwritable-expected...` comments in `root_cause.c`.

This is mentioned in [the Clang diagnostic verifier documentation](https://clang.llvm.org/doxygen/classclang_1_1VerifyDiagnosticConsumer.html#details), which is linked from [the 3C diagnostic verification documentation](https://github.com/correctcomputation/checkedc-clang/blob/main/clang/docs/checkedc/3C/CONTRIBUTING.md#diagnostic-verification).  Can you think of a way to make it more obvious for our team?